unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatWindowController.cs

```diff
@@ -77,6 +81,12 @@ public void Initialize(IPublicChatWindowView view = null, bool isVisible = true)
             chatController.OnChannelUpdated += HandleChannelUpdated;
 
             dataStore.mentions.someoneMentionedFromContextMenu.OnChange += SomeoneMentionedFromContextMenu;
+            dataStore.player.otherPlayers.OnAdded += UpdateMembersCount;
```

Comment:
Maybe introduce a property to wrap frequently accessed `dataStore.player.otherPlayers` ?